components/load-more-button.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function LoadMoreButton(props) {
+    return <button styleName={props.styleName} onClick={props.onLoadMore}>Load more</button>
+}
+
+export { LoadMoreButton };
```

Comment:
not sure, will this work? Whenever babel encounters `styleName` property, it tries to find the corresponding module which has the respective `styleName` property. In this case it'll be undefined?